app/src/components/Graph.js

```diff
@@ -157,13 +159,17 @@ const Graph = ({
           // with the same index. Is not critical but it's a bug
           index: 0
         },
-        ...entries,
-        {
+        ...entries
+      ];
+
```

Comment:
This is not fully true. From my point of view if last trade is done for example a week before resolution date, once we surpass the resolution date the graph will show the last trade date as final date and this can be felt like an error.

Logic here should be a bit more complex and I would not add this last entry only if we have a trade that was executed "after" the resolution date. There are few cases where this can happen. Other than that I would always add this entry but selecting if I want to assign `currentDate` or `resolutionDate`

Reply:
The last trade is infact not a "last trade" but the *current probability* it will always have the date of `now` and the probabilities that the market as at this moment.

Reply:
It will have these values (current probability and current date/time)
```
          outcomesProbability: currentProbability,
          date: +new Date(),
```

Reply:
This was done so that the graph does not suddenly end, but now we kind of want that, once the markets been resolved :)

Reply:
I also understand what you mean. So in case market is closed we should duplicate last trade but setting the date to the closing date, so it makes clear that this was market final value at the closing date.